src/limestone-api.ts

```diff
@@ -407,20 +407,7 @@ export default class LimestoneApi {
     // TODO: Maybe we can pass the signed string in broadcaster
     // to avoid potential problems with signature verification
 
-    // It is important to have properties of price object in the
-    // same order as they have been set before signing
-    const signedData = JSON.stringify(
-      _.pick(price, [
-        "id",
-        "source",
-        "symbol",
-        "timestamp",
-        "version",
-        "value",
-        "permawebTx",
-        "provider",
-      ]),
-    );
+    const signedData = this.getPriceSignedData(price);
     const publicKey = String(price.providerPublicKey);
```

Comment:
Hmm, can we have PriceDataWithSignature without publicKey, does it make sense to implement this interface and get signature but without the key?